code/modules/mob/living/living.dm

```diff
@@ -192,10 +192,13 @@
 		var/current_dir
 		if(isliving(AM))
 			current_dir = AM.dir
-		step(AM, t)
+		if(step(AM, t))
+			step(src, t)
 		if(current_dir)
 			AM.setDir(current_dir)
-		now_pushing = 0
+		now_pushing = FALSE
+	else if(move_force > AM.move_resist)
+		force_push(AM)
```

Comment:
`move_force` and `move_resist` need to replace anchored in mob entirely. As it is here this is only affecting things which are anchored.

Reply:
kay